compose/foundation/foundation/src/jvmMain/kotlin/androidx/compose/foundation/text/Actuals.jvm.kt

```diff
@@ -16,14 +16,13 @@
 
 package androidx.compose.foundation.text
 
-import org.jetbrains.skiko.OS
-import org.jetbrains.skiko.hostOs
+import androidx.compose.foundation.text.selection.TextFieldSelectionManager
+import androidx.compose.runtime.Composable
 
-internal actual val platformDefaultKeyMapping: KeyMapping = createPlatformDefaultKeyMapping(hostOs)
-
-internal fun createPlatformDefaultKeyMapping(platform: OS): KeyMapping {
-    return when (platform) {
-        OS.MacOS -> createMacosDefaultKeyMapping()
-        else -> defaultKeyMapping
-    }
+@Composable
```

Comment:
Note: github is displaying this incorrectly. File `KeyMapping.jsMain.kt` was renamed into `KeyMapping.web.kt`.

But file `Actuals.jvm.kt` is actually a new file.